src/commands/backlog/checkin.ts

```diff
@@ -0,0 +1,64 @@
+import {GitBaseCommand} from '../../git-base'
+import {flags} from '@oclif/command'
+import {AxiosJiraClient} from '../../api/service/axios-jira-client'
+import {AxiosBitBucketClient} from '../../api/service/axios-bitbucket-client'
+
+export default class GitPush extends GitBaseCommand {
+  static description = 'Push local changes (commits) to the remote repository'
+
+  static flags = {
+    pr: flags.boolean({description: 'Create Pull-Request', default: true}),
+  }
+
+  async run() {
```

Comment:
My eyes are defocusing looking at this. This should be split into multiple function calls to isolate and describe the steps going on here.

Reply:
For future, it may be easier to enforce lines of a function as part of eslint rule.